gapic/src/main/com/google/gapid/views/CommandTree.java

```diff
@@ -312,6 +312,12 @@ protected boolean isLoaded(CommandStream.Node element) {
           return element.getData() != null;
         }
 
+        @Override
+        protected boolean isDefaultExpanded(CommandStream.Node element) {
+          Service.CommandTreeNode data = element.getData();
+          return data != null && data.getExpandByDefault();
```

Comment:
We always check if the node is expanded by default even though we're hardcoding true for every node in the command tree in the Go file. I guess we want to have the freedom to control this from the Go side in the future?

Reply:
We only hard code it for `SubCmdRoot` nodes, not any of the other grouping nodes. So some groups will indeed have false for the UI (Command buffers and below).